README.md

```diff
@@ -10,20 +10,18 @@ Please check our [full documentation](https://flax.readthedocs.io/) website to l
 
 **NOTE**: Flax is in use by a growing community
 of researchers and engineers at Google who happily use Flax for their
-daily research. The new Flax ["Linen" module API](https://github.com/google/flax/tree/master/flax/linen) is now stable and we recommend it for all new projects. The old `flax.nn` API will be deprecated. Please report
+daily research. The new Flax ["Linen" module API](https://github.com/google/flax/tree/master/flax/linen/README.md) is now stable and we recommend it for all new projects. The old `flax.nn` API will be deprecated. Please report
```

Comment:
A general remark : I'm a big fan of wrapped paragraphs.

One of many advantages is that `vscode` side by side review is much easier when paragraphs are wrapped (due to https://github.com/microsoft/vscode/issues/11387 )

Reply:
Done